urlx.go

```diff
@@ -62,7 +62,7 @@ func defaultScheme(rawURL, scheme string) string {
 }
 
 var (
-	domainRegexp = regexp.MustCompile(`^([a-zA-Z0-9-]{1,63}\.)*([a-zA-Z0-9-]{1,63})$`)
+	domainRegexp = regexp.MustCompile(`^([a-zA-Z0-9-_]{1,63}\.)*([a-zA-Z0-9-]{1,63})$`)
```

Comment:
imho, we shouldn't allow the underscores in the top level domains -- we should allow it in subdomains only

is that correct?

Reply:
yes